src/tribler/core/sentry_reporter/sentry_reporter.py

```diff
@@ -280,7 +280,7 @@ def event_from_exception(self, exception) -> Dict:
         Returns:
             the event that has been saved in `_before_send` method
         """
-        self._logger.info(f"Event from exception: {exception}")
+        self._logger.debug(f"Event from exception: {exception}")
```

Comment:
To avoid spamming logs with multiple copies of the same error traceback

Reply:
NIT. As an alternative option, you can use `SentryReporter.ignore_logger()`

Reply:
As I understand it, `SentryReporter.ignore_logger()` affects what we send to Sentry, but does not affect our log files.